src/Microsoft.DotNet.Darc/src/Darc/Operations/AddDefaultChannelOperation.cs

```diff
@@ -24,7 +24,13 @@ public override async Task<int> ExecuteAsync()
         {
             try
             {
-                IRemote remote = RemoteFactory.GetBarOnlyRemote(_options, Logger);
+                IRemote remote = RemoteFactory.GetRemote(_options, _options.Repository, Logger);
+                
+                if (!(await UxHelpers.VerifyMaestroManagedBranchExists(remote, _options.Repository, _options.Branch, !_options.NoConfirmation)))
+                {
+                    Console.WriteLine("Aborting default channel creation.");
```

Comment:
NIT: May be add the reason.

Reply:
The VerifyMaestroManagedBranchExists call itself will have the reason. I think these messages will have enough info.